baseline/vectorizers.py

```diff
@@ -1524,10 +1524,17 @@ def __init__(self, **kwargs):
         )
 
         self.mxlen = kwargs.get('mxlen', -1)
-        Offsets.INDICES['PAD'] = self.vocab['<pad>']
-        Offsets.INDICES['GO'] = self.vocab['<s>']
-        Offsets.INDICES['EOS'] = self.vocab['</s>']
-        Offsets.INDICES['UNK'] = self.vocab['<unk>']
+
+        if '<pad>' in self.vocab:
```

Comment:
is there any code that actually exercises this path right now?

Reply:
These were the changes from closed PR (https://github.com/dpressel/mead-baseline/pull/927/files#diff-2c3874d9f72891a8ff64d23255bc37111b8dfda619baead4b0b02a4a73c77495R1527) to facilitate GPT2 completer i believe